open-class/docs/classes/assignment-branch-logic.md

```diff
@@ -10,244 +10,187 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
-
-## Building our Docassemble interview
 
-### What kinds of information do we need to gather?
-
-Before we start, let's think about what kinds of information we will want to ask.
-
-This Docassemble interview will use two kinds of screens:
-
-1. Yes/no questions
-2. Ending screens
-
-As a reminder, ending screens use the `event` specifier. 
-
-We haven't seen a `question` block that uses just a yes/no question yet. Here's a [link to the documentation](https://docassemble.org/docs/questions.html#tocAnchor-1-6-1).
-
-Let's take a look at an example:
+We'll need two ending screens. You make endings screens with an [`event` attribute](https://docassemble.org/docs/questions.html#event) in a `question` block. That's like the `field` attribute we've used before. Unlike regular question screens, your user isn't going to give an answer here, so there's no variable for Docassemble to find. The `event` attribute tells Docassemble how to find the question without needing a variable name.
 
 ```yaml
+---
+id: no_go_advice
+event: dont_go_to_party
+question: |
+  Don't go to that dumb party. Do something fun instead!
+---
+id: go_advice
+event: go_to_party
 question: |
-  Are you at least 18 years of age?
-yesno: over_eighteen
+  You should go to the party! I promise it'll be awesome!
+---
 ```
 
-And here's what it looks like when we run it:
-
-<img alt="Yes/No question" src={useBaseUrl('img/Yes_no.png')} />
+### What do you need to know?
 
-Docassemble uses two different kinds of `question` blocks. This new kind only lets you ask for one variable on a screen, and doesn't use the `fields` specifier we learned earlier. It's kind of a simplified shorcut version.
+Well, without user research, you don't really know what good questions would be. You don't have funding or time to do that discovery work, though, but you don't care that much about those strangers and it's not like they have other options, so you'll start by making a guess and worry about it later.
 
-Let's look at line 3 of our sample:
+Do they even like parties? That seems like a place to start.
 
 ```yaml
-yesno: over_eighteen
+---
+id: does_like_parties
+question: |
+  Do you like parties?
+fields:
+  - Do you like parties: user_likes_parties
+---
 ```
+<!-- I'd stick with fields for now because those will be used most consistently -->
 
-This is a new specifier: the `yesno` specifier. It always shows the text `yes` and `no` as buttons on the screen. On the right hand side of the : is the variable that will be assigned the value `True` or `False`, depending on which button the user clicks.
-
-### Planning out our variables
-
-It can help to start out by writing a list of the actual variables we will want to ask. This way, we can plan variable names that make sense together.
-
-In Docassemble interviews, it's traditional to use variables names that are:
```

Comment:
Note: I like the idea of this lesson in general, but it's another concept and so I chose to leave it out for this lesson. Since Assignment 1 didn't have a chance to deal with it this time around, I might make it supplementary material or extra credit. Also, I think there are better reasons we can site for clear names than tradition, and the reasons shown below are actually split between two purposes:

1. How to choose good names themselves
1. The formatting conventions specific to Python